Project.toml

```diff
@@ -0,0 +1,36 @@
+name = "OrdinaryDiffEq"
+uuid = "9c2ba320-684e-11e9-1325-35842128880e"
```

Comment:
This needs to be https://github.com/JuliaRegistries/General/blob/5779b981069587d8e933415ab7f8b71a51151c1b/O/OrdinaryDiffEq/Package.toml#L2, xref https://github.com/JuliaLang/julia/issues/31859